src/Server/RestRouterHandlers/TableRestRouterHandler.cpp

```diff
@@ -277,6 +290,24 @@ String TableRestRouterHandler::getCreationSQL(const Poco::JSON::Object::Ptr & pa
         create_segments.push_back(", shard=" + shard);
     }
 
+    String create_table_settings = "";
+    for (const auto & setting : CREATE_TABLE_SETTINGS)
+    {
+        if (hasQueryParameter(setting))
+        {
+            const String & setting_str = setting + "=" + getQueryParameter(setting);
+            create_segments.push_back(", " + setting_str);
+
+            if (!create_table_settings.empty())
+            {
+                create_table_settings += "&";
```

Comment:
Suggest to use boost::algorithm::join() here

Reply:
Good suggestion